extension.html

```diff
@@ -114,7 +145,7 @@ <h2>Extensions to the <code>PointerEvent</code> interface</h2>
 
     <section>
       <h2>Timing of coalesced event firing</h2>
-      <p>Coalescing events only happens for <code>pointermove</code> events. All the coalesced events should be fired before the next frame or before any <code>pointerup</code>, <code>pointerdown</code>, <code>pointercancel</code> events. Note that user agents don't need to hit-test all the coalesced events as none of them will have null targets.</p>
+      <p>Coalescing events only happens for <code>pointermove</code> events. All the coalesced events should be fired before the next frame or before any <code>pointerup</code>, <code>pointerdown</code>, <code>pointercancel</code> events. Note that user agents don't need to hit-test all the coalesced events as all of them will have the same target as the dispatched event.</p>
```

Comment:
... especially when this talks about coalescing happening for pointermove events only.

Reply:
Fair. I forgot to update this. Particularly I already talked about the dispatch time of these coalesced events before. I removed that sentence from this paragraph.